### DISCUSSION TITLE
Listen to postgressql notification event

### DISCUSSION THREAD

**Discussion by kkhanhluu** [mikro-orm/mikro-orm#2285] [Q&A] [answered]
Is it possible to listen to `notification` event of postgres? With [node-postgres](https://node-postgres.com/api/client) we can add a call back to listen to postgres's [listen/notify](https://www.postgresql.org/docs/9.1/sql-notify.html). Can we achieve that behavior with `mikro-orm`?

Many thanks!

**Comment by B4nan** [comment, marked as answer]
If it's possible to do it via knex, then you could use `driverOptions` to adjust its config. If not, you could access the pg client too, with typecasting or ts-ignore comments - the knex instance is on `Connection.client`, you will need to look up where knex is storing the pg connector instance itself.

```ts
const connection = em.getConnection();
// @ts-ignore
const knexClient = connection.client;
// @ts-ignore
const pg = knexClient...
```

**Reply by kkhanhluu** [reply]
@B4nan thank you, it worked. That's my solution in case someone needs that in the future: 
```javascript
  const mikroOrmConnection = em.getConnection();
  // @ts-ignore
  const knexClient = mikroOrmConnection.client;
  const connection = await knexClient.client.acquireConnection();
  connection.query(`LISTEN ${your_event}`);
  connection.on('notification',  (msg) => {
    console.log(msg);
  });
```
